mitodl/open-discussions
# Comments pagination

#### What are the relevant tickets?
Fixes #78 

#### What's this PR do?
Implements the comment pagination

#### How should this be manually tested?
Add many comments to a post, such that when you see it in reddit you are not shown all of them at once. In open-discussions click on the 'Load more comments' link. You should see the link disappear and be replaced with some extra comments
